src/main/java/no/unit/nva/cristin/institutions/PresentationConverter.java

```diff
@@ -0,0 +1,56 @@
+package no.unit.nva.cristin.institutions;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.TreeMap;
+
+public class PresentationConverter {
+
+    protected InstitutionPresentation asInstitutionPresentation(Institution institution) {
+        InstitutionPresentation institutionPresentation = new InstitutionPresentation();
+        institutionPresentation.cristinInstitutionId = institution.cristinInstitutionId;
+
+        Optional.ofNullable(institution.institutionName).orElse(new TreeMap<String, String>() {
+        }).forEach((key, value) -> {
+            NamePresentation namePresentation = new NamePresentation();
+            namePresentation.language = key;
+            namePresentation.name = value;
+            institutionPresentation.institutionNames.add(namePresentation);
+        });
+
+        institutionPresentation.acronym = Optional.ofNullable(institution.acronym).orElse("");
+        institutionPresentation.country = Optional.ofNullable(institution.country).orElse("");
```

Comment:
Use constant EMPTY_STRING
```suggestion
        institutionPresentation.acronym = Optional.ofNullable(institution.acronym).orElse(EMPTY_STRING);
        institutionPresentation.country = Optional.ofNullable(institution.country).orElse(EMPTY_STRING);
```